{"uuid": "ab90b75d-5865-44fd-b05a-c85bf38f94ff", "counter": 96, "stdout": "", "start_line": 1154, "end_line": 1154, "runner_ident": "93a1b50d-a222-430e-906f-9a2067fef8e9", "event": "runner_on_failed", "pid": 25, "created": "2023-04-07T08:00:56.763276", "parent_uuid": "0242ac16-0009-4230-e26c-00000000000b", "event_data": {"playbook": "/data/project/playbooks/step02.yml", "playbook_uuid": "e39471e7-eb66-4c95-8d3c-fa7913bab9b5", "play": "localhost", "play_uuid": "0242ac16-0009-4230-e26c-000000000008", "play_pattern": "localhost", "task": "generate config", "task_uuid": "0242ac16-0009-4230-e26c-00000000000b", "task_action": "template", "resolved_action": "ansible.builtin.template", "task_args": "", "task_path": "/data/project/playbooks/step02.yml:50", "host": "localhost", "remote_addr": "localhost", "res": {"results": [{"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.0.0/30", "agent_name": "Seg-192.168.0.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth1.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth1.0", "original_if": "regiona-rt1_ge-0/0/0.0"}, {"name": "regiona-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regiona-rt2_eth1.0", "original_if": "regiona-rt2_ge-0/0/0.0"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.0.0/30", "agent_name": "Seg-192.168.0.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth1.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth1.0", "original_if": "regiona-rt1_ge-0/0/0.0"}, {"name": "regiona-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regiona-rt2_eth1.0", "original_if": "regiona-rt2_ge-0/0/0.0"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regiona-rt1", "agent_name": "regiona-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.0.1/30", "description": "to_Seg-192.168.0.0-30_Ethernet1", "original_if": "ge-0/0/0.0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "172.16.0.1/30", "description": "to_Seg-172.16.0.0-30_Ethernet1", "original_if": "ge-0/0/1.0"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "172.16.1.1/30", "description": "to_Seg-172.16.1.0-30_Ethernet1", "original_if": "ge-0/0/2.0"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regiona-rt1", "agent_name": "regiona-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.0.1/30", "description": "to_Seg-192.168.0.0-30_Ethernet1", "original_if": "ge-0/0/0.0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "172.16.0.1/30", "description": "to_Seg-172.16.0.0-30_Ethernet1", "original_if": "ge-0/0/1.0"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "172.16.1.1/30", "description": "to_Seg-172.16.1.0-30_Ethernet1", "original_if": "ge-0/0/2.0"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regiona-rt2", "agent_name": "regiona-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.0.2/30", "description": "to_Seg-192.168.0.0-30_Ethernet2", "original_if": "ge-0/0/0.0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.100.1/24", "description": "to_Seg-192.168.100.0-24_Ethernet1", "original_if": "ge-0/0/1.0"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regiona-rt2", "agent_name": "regiona-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.0.2/30", "description": "to_Seg-192.168.0.0-30_Ethernet2", "original_if": "ge-0/0/0.0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.100.1/24", "description": "to_Seg-192.168.100.0-24_Ethernet1", "original_if": "ge-0/0/1.0"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_172.16.0.0/30", "agent_name": "Seg-172.16.0.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth2.0", "original_if": "regiona-rt1_ge-0/0/1.0"}, {"name": "regionb-rt1_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt1_eth1.0", "original_if": "regionb-rt1_Ethernet2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_172.16.0.0/30", "agent_name": "Seg-172.16.0.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth2.0", "original_if": "regiona-rt1_ge-0/0/1.0"}, {"name": "regionb-rt1_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt1_eth1.0", "original_if": "regionb-rt1_Ethernet2"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionb-rt1", "agent_name": "regionb-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "172.16.0.2/30", "description": "to_Seg-172.16.0.0-30_Ethernet2", "original_if": "Ethernet2"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "172.16.2.1/30", "description": "to_Seg-172.16.2.0-30_Ethernet2", "original_if": "Ethernet3"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "192.168.1.1/30", "description": "to_Seg-192.168.1.0-30_Ethernet1", "original_if": "Ethernet1"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionb-rt1", "agent_name": "regionb-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "172.16.0.2/30", "description": "to_Seg-172.16.0.0-30_Ethernet2", "original_if": "Ethernet2"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "172.16.2.1/30", "description": "to_Seg-172.16.2.0-30_Ethernet2", "original_if": "Ethernet3"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "192.168.1.1/30", "description": "to_Seg-192.168.1.0-30_Ethernet1", "original_if": "Ethernet1"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_172.16.1.0/30", "agent_name": "Seg-172.16.1.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth3.0", "original_if": "regiona-rt1_ge-0/0/2.0"}, {"name": "regionc-rt1_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-rt1_eth1.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/1"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_172.16.1.0/30", "agent_name": "Seg-172.16.1.0-30", "type": "segment", "if_list": [{"name": "regiona-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt1_eth3.0", "original_if": "regiona-rt1_ge-0/0/2.0"}, {"name": "regionc-rt1_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-rt1_eth1.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/1"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionc-rt1", "agent_name": "regionc-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "172.16.1.2/30", "description": "to_Seg-172.16.1.0-30_Ethernet2", "original_if": "GigabitEthernet0/0/0/1"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.2.1/30", "description": "to_Seg-192.168.2.0-30_Ethernet1", "original_if": "GigabitEthernet0/0/0/0"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "172.16.2.2/30", "description": "to_Seg-172.16.2.0-30_Ethernet1", "original_if": "GigabitEthernet0/0/0/2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionc-rt1", "agent_name": "regionc-rt1", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "172.16.1.2/30", "description": "to_Seg-172.16.1.0-30_Ethernet2", "original_if": "GigabitEthernet0/0/0/1"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.2.1/30", "description": "to_Seg-192.168.2.0-30_Ethernet1", "original_if": "GigabitEthernet0/0/0/0"}, {"name": "eth3.0", "agent_name": "eth3.0", "ipv4": "172.16.2.2/30", "description": "to_Seg-172.16.2.0-30_Ethernet1", "original_if": "GigabitEthernet0/0/0/2"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.2.0/30", "agent_name": "Seg-192.168.2.0-30", "type": "segment", "if_list": [{"name": "regionc-rt1_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt1_eth2.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/0"}, {"name": "regionc-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-rt2_eth1.0", "original_if": "regionc-rt2_GigabitEthernet0/0/0/0"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.2.0/30", "agent_name": "Seg-192.168.2.0-30", "type": "segment", "if_list": [{"name": "regionc-rt1_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt1_eth2.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/0"}, {"name": "regionc-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-rt2_eth1.0", "original_if": "regionc-rt2_GigabitEthernet0/0/0/0"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionc-rt2", "agent_name": "regionc-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.2.2/30", "description": "to_Seg-192.168.2.0-30_Ethernet2", "original_if": "GigabitEthernet0/0/0/0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.100.1/25", "description": "to_Seg-192.168.100.0-25_Ethernet1", "original_if": "GigabitEthernet0/0/0/1"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionc-rt2", "agent_name": "regionc-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.2.2/30", "description": "to_Seg-192.168.2.0-30_Ethernet2", "original_if": "GigabitEthernet0/0/0/0"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.100.1/25", "description": "to_Seg-192.168.100.0-25_Ethernet1", "original_if": "GigabitEthernet0/0/0/1"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_172.16.2.0/30", "agent_name": "Seg-172.16.2.0-30", "type": "segment", "if_list": [{"name": "regionc-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt1_eth3.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/2"}, {"name": "regionb-rt1_eth2.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt1_eth2.0", "original_if": "regionb-rt1_Ethernet3"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_172.16.2.0/30", "agent_name": "Seg-172.16.2.0-30", "type": "segment", "if_list": [{"name": "regionc-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt1_eth3.0", "original_if": "regionc-rt1_GigabitEthernet0/0/0/2"}, {"name": "regionb-rt1_eth2.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt1_eth2.0", "original_if": "regionb-rt1_Ethernet3"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.100.0/25", "agent_name": "Seg-192.168.100.0-25", "type": "segment", "if_list": [{"name": "regionc-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt2_eth2.0", "original_if": "regionc-rt2_GigabitEthernet0/0/0/1"}, {"name": "regionc-svr01_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-svr01_eth1.0", "original_if": "regionc-svr01_ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.100.0/25", "agent_name": "Seg-192.168.100.0-25", "type": "segment", "if_list": [{"name": "regionc-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionc-rt2_eth2.0", "original_if": "regionc-rt2_GigabitEthernet0/0/0/1"}, {"name": "regionc-svr01_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionc-svr01_eth1.0", "original_if": "regionc-svr01_ens2"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionc-svr01", "agent_name": "regionc-svr01", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.100.10/25", "description": "to_Seg-192.168.100.0-25_Ethernet2", "original_if": "ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionc-svr01", "agent_name": "regionc-svr01", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.100.10/25", "description": "to_Seg-192.168.100.0-25_Ethernet2", "original_if": "ens2"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.1.0/30", "agent_name": "Seg-192.168.1.0-30", "type": "segment", "if_list": [{"name": "regionb-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionb-rt1_eth3.0", "original_if": "regionb-rt1_Ethernet1"}, {"name": "regionb-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt2_eth1.0", "original_if": "regionb-rt2_Ethernet1"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.1.0/30", "agent_name": "Seg-192.168.1.0-30", "type": "segment", "if_list": [{"name": "regionb-rt1_eth3.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionb-rt1_eth3.0", "original_if": "regionb-rt1_Ethernet1"}, {"name": "regionb-rt2_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-rt2_eth1.0", "original_if": "regionb-rt2_Ethernet1"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionb-rt2", "agent_name": "regionb-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.1.2/30", "description": "to_Seg-192.168.1.0-30_Ethernet2", "original_if": "Ethernet1"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.254.1/24", "description": "to_Seg-192.168.254.0-24_Ethernet1", "original_if": "Ethernet2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionb-rt2", "agent_name": "regionb-rt2", "type": "node", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.1.2/30", "description": "to_Seg-192.168.1.0-30_Ethernet2", "original_if": "Ethernet1"}, {"name": "eth2.0", "agent_name": "eth2.0", "ipv4": "192.168.254.1/24", "description": "to_Seg-192.168.254.0-24_Ethernet1", "original_if": "Ethernet2"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.254.0/24", "agent_name": "Seg-192.168.254.0-24", "type": "segment", "if_list": [{"name": "regionb-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionb-rt2_eth2.0", "original_if": "regionb-rt2_Ethernet2"}, {"name": "regionb-svr01_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-svr01_eth1.0", "original_if": "regionb-svr01_ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.254.0/24", "agent_name": "Seg-192.168.254.0-24", "type": "segment", "if_list": [{"name": "regionb-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regionb-rt2_eth2.0", "original_if": "regionb-rt2_Ethernet2"}, {"name": "regionb-svr01_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regionb-svr01_eth1.0", "original_if": "regionb-svr01_ens2"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regionb-svr01", "agent_name": "regionb-svr01", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.254.10/24", "description": "to_Seg-192.168.254.0-24_Ethernet2", "original_if": "ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regionb-svr01", "agent_name": "regionb-svr01", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.254.10/24", "description": "to_Seg-192.168.254.0-24_Ethernet2", "original_if": "ens2"}]}}, {"changed": false, "skipped": true, "skip_reason": "Conditional result was False", "_ansible_no_log": false, "item": {"name": "Seg_192.168.100.0/24", "agent_name": "Seg-192.168.100.0-24", "type": "segment", "if_list": [{"name": "regiona-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt2_eth2.0", "original_if": "regiona-rt2_ge-0/0/1.0"}, {"name": "regiona-svr02_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regiona-svr02_eth1.0", "original_if": "regiona-svr02_ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "Seg_192.168.100.0/24", "agent_name": "Seg-192.168.100.0-24", "type": "segment", "if_list": [{"name": "regiona-rt2_eth2.0", "agent_name": "Ethernet1", "ipv4": null, "description": "to_regiona-rt2_eth2.0", "original_if": "regiona-rt2_ge-0/0/1.0"}, {"name": "regiona-svr02_eth1.0", "agent_name": "Ethernet2", "ipv4": null, "description": "to_regiona-svr02_eth1.0", "original_if": "regiona-svr02_ens2"}]}}, {"failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'", "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1156, in do_template\n    res = j2_concat(rf)\n  File \"<template>\", line 16, in root\n  File \"/usr/lib/python3.8/site-packages/jinja2/runtime.py\", line 747, in _fail_with_undefined_error\n    raise self._undefined_exception(self._undefined_message)\njinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iflist'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/lib/python3.8/site-packages/ansible/plugins/action/template.py\", line 150, in run\n    resultant = templar.do_template(template_data, preserve_trailing_newlines=True, escape_backslashes=False)\n  File \"/usr/local/lib/python3.8/site-packages/ansible/template/__init__.py\", line 1193, in do_template\n    raise AnsibleUndefinedVariable(e)\nansible.errors.AnsibleUndefinedVariable: 'dict object' has no attribute 'iflist'\n", "_ansible_no_log": false, "changed": false, "item": {"name": "regiona-svr02", "agent_name": "regiona-svr02", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.100.210/24", "description": "to_Seg-192.168.100.0-24_Ethernet2", "original_if": "ens2"}]}, "ansible_loop_var": "item", "_ansible_item_label": {"name": "regiona-svr02", "agent_name": "regiona-svr02", "type": "endpoint", "if_list": [{"name": "eth1.0", "agent_name": "eth1.0", "ipv4": "192.168.100.210/24", "description": "to_Seg-192.168.100.0-24_Ethernet2", "original_if": "ens2"}]}}], "msg": "One or more items failed", "changed": false}, "start": "2023-04-07T08:00:55.995981", "end": "2023-04-07T08:00:56.762950", "duration": 0.766969, "ignore_errors": null, "event_loop": null, "uuid": "ab90b75d-5865-44fd-b05a-c85bf38f94ff"}}